README.md

```diff
@@ -0,0 +1,35 @@
+# My-Portfolio
+
+# Project Name
+My Portfolio Project is basic web app to showcase my skills, experince and projects that I have done. The website is made with HTML/CSS and linters to improve code quality. The project can be used as a template for static web app projects that implement github workflows
+
+
+## Built With
+
+- HTML/CSS
+- linters
+
+## Getting Started
+
```

Comment:
- [ ] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
 So that you can have this section as part of your Readme
![Capturereadme](https://user-images.githubusercontent.com/81444338/170479126-84df6ce7-3e3a-4045-be0d-7142a08fccf8.PNG)